docs/api/python/frozen/pyopencolorio_cpuprocessor.rst

```diff
@@ -30,14 +30,14 @@
 
       Overloaded function.
 
-      1. applyRGB(self: PyOpenColorIO.CPUProcessor, pixel: buffer) -> buffer
+      1. applyRGB(self: PyOpenColorIO.CPUProcessor, data: buffer) -> None
 
       Apply to a single pixel respecting that the input and output bit-depths be 32-bit float and the image buffer be packed RGB/RGBA.
```

Comment:
"Apply to a single pixel ..." needs to get updated now, could be more than that, right?

Also, does the note about efficiency still apply?  Maybe needs to be reworded to encourage passing in multiple pixels rather than looping in python.

Reply:
I updated the documentation in the Python binding source to explain each of the apply methods and noting where they differ in functionality from the C++.